lib/parser-stats.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const promisify = require('bluebird').promisify;
+const Table = require('cli-table');
+const glob = promisify(require('glob').glob);;
+const filesize = require('filesize');
+const percent = require('calc-percent');
+const fs = require('fs');
+
+
+var table = new Table({
```

Comment:
`table =` is assigned a pointer to an object. The pointer doesn't change after this. So you can use `const table =`.